Sources/Controllers/ALKConversationViewController.swift

```diff
@@ -557,6 +562,15 @@ open class ALKConversationViewController: ALKBaseViewController, Localizable {
         contextTitleView.configureWith(value: topicDetail)
         toggleVisibilityOfContextTitleView(true)
     }
+    
+    public func prepareConversationInfoView() {
+        guard let conversationInfoModel = configuration.conversationInfoModel else { return }
+        conversationInfoView.configureViewWith(model: conversationInfoModel)
+    }
+    
+    @objc func conversationInfoViewTap(_: UITapGestureRecognizer) {
+        ALKCustomEventHandler.shared.publish(triggeredEvent: CustomEvent.conversationInfoClick, data:  nil)
```

Comment:
Add this as well in documentation.

Reply:
Will add it on our documentation while releasing this